cranelift/filetests/src/test_compile.rs

```diff
@@ -39,6 +39,8 @@ impl SubTest for TestCompile {
     fn run(&self, func: Cow<ir::Function>, context: &Context) -> SubtestResult<()> {
         let isa = context.isa.expect("compile needs an ISA");
         let mut comp_ctx = cranelift_codegen::Context::for_function(func.into_owned());
+        // With `MachBackend`s, we need to explicitly request dissassembly results.
+        comp_ctx.set_disasm(true);
```

Comment:
Should this be guarded by `if isa.get_mach_backend().is_some() ...` like below then?

Reply:
Fixed, thanks!